pay-api/src/pay_api/services/cfs_service.py

```diff
@@ -49,6 +49,41 @@ def create_cfs_account(cls, name: str, contact_info: Dict[str, Any],
 
         return account_details
 
+    @staticmethod
+    def validate_bank_account(bank_details: Tuple[Dict[str, Any]]):
```

Comment:
Better to start adding correct return types. I know we were not following this, but for new functions we could start doing this.